docs/installation.rst

```diff
@@ -1,86 +1,83 @@
+************
 Installation
-============
+************
 
-Installing composer is quick and simple.
+Installing via composer is quick and simple, and the plugin is `available on Packagist <https://packagist.org/packages/friendsofcake/crud>`_.
```

Comment:
What I've  done for other (e.g. muffin) plugins is combine the text with the links. Something like:

The recommended installation method for this plugin is by installing the [foc/crud package](link-to-composer) using [composer](link-to-get-composer).